### ISSUE TITLE
[stable/spinnaker] Fails because of minio container issue

### ISSUE THREAD

**Issue by mdagost** [helm/charts#11213]
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
Tested on both Minikube v0.33.1 and GKE

**Which chart**:
spinnaker

**What happened**:
`helm install -n kubelive stable/spinnaker -f values.yaml` with the default values.yaml fails because it can't pull the minio container `RELEASE.2018-06-09T02-18-09Z`.  There is indeed no image with that tag on dockerhub.

**What you expected to happen**:
It shouldn't fail

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by huerlisi** [comment]
Newest `values.yaml` uses `RELEASE.2019-02-13T19-48-27Z` which works properly.

**Comment by ChrisJBurns** [comment]
I am having the same problem tonight and it cannot find the image. I used an other image locally and it worked fine. You say that it has been updated, but in the following values.yaml: https://github.com/helm/charts/blob/master/stable/spinnaker/values.yaml
It is still `RELEASE.2018-06-09T02-18-09Z` and not `RELEASE.2019-02-13T19-48-27Z`

**Comment by ChrisJBurns** [comment]
This is an issue I believe. @huerlisi

**Comment by ChrisJBurns** [comment]
Not sure why this is closed as it is still a problem. I have put fix in PR #13968

**Comment by deepforu47** [comment]
I am still getting error that image is not found.

```
kubelet, minikube  Failed to pull image "minio/minio:RELEASE.2019-02-13T19-48-27Z": rpc error: code = Unknown desc = Error response from daemon: manifest for minio/minio:RELEASE.2019-02-13T19-48-27Z not found
```
